x-pack/test/epm_api_integration/config.ts

```diff
@@ -26,11 +26,7 @@ export default async function({ readConfigFile }: FtrConfigProviderContext) {
 
     kbnTestServer: {
       ...xPackAPITestsConfig.get('kbnTestServer'),
-      serverArgs: [
-        ...xPackAPITestsConfig.get('kbnTestServer.serverArgs'),
-        '--xpack.ingestManager.epm.enabled=true',
```

Comment:
@elastic/ingest-management I moved these to the top level `api_integration/config.js` file because the endpoint app's configuration is there. I guess another option would be to create a new `config.js` file for endpoint if we did not want `epm` enabled by default in the kibana tests.